### PR TITLE
System tests: adjust JSON expectations to reality

### PR DESCRIPTION
**Pull request by nLight**
I've changed the way we remove empty values from JSON our expectations went broken. This PR fixes that.

### CODE REVIEW

File: system-tests/services/test-pods.js [hunk 1]
```diff
@@ -1887,7 +1942,14 @@ describe("Services", function() {
...
             }
-          ]
+          ],
+          "volumes": [],
+          "fetch": [],
+          "scheduling": {
+            "placement": {
+              "constraints": []
+            }
+          }
```
Comment (tnguyen0) [inline system-tests/services/test-pods.js]: No quotes :)

Comment (nLight) [inline system-tests/services/test-pods.js]: Please expand the piece of code it belongs to. In this particular place we actually need quotes.

**Comment by nLight** [comment]
I bet those are failing because of #2295 I will rebase and see if it resolves the issue.

**Comment by orlandohohmeier** [comment]
LGTM 💯 ✨

File: system-tests/services/test-pods.js (hunk 1 above)
Comment (tnguyen0) [inline system-tests/services/test-pods.js]: You are correct!

**Pull request merged**